sdk/storage/inc/common/constants.hpp

```diff
@@ -20,4 +20,7 @@ namespace Azure { namespace Storage { namespace Details {
   constexpr static const char* c_defaultSasVersion = "2019-12-12";
 
   constexpr int c_reliableStreamRetryCount = 3;
+  constexpr int64_t c_FileUploadDefaultChunkSize = 4 * 1024 * 1024;
+  constexpr int64_t c_DownloadDefaultChunkSize = 4 * 1024 * 1024;
```

Comment:
why do we define these variables here? Are they going to be used at more than once place?

Reply:
`c_FileUploadDefaultChunkSize` is used in two functions and `c_DownloadDefaultChunkSize` is used in four functions. I think it should be defined in a place for an easier change in the future.

Reply:
which four functions?

Reply:
`c_DownloadDefaultChunkSize ` should be used in Blob's concurrent download and File's concurrent download, unless there is a good reason for them to have different chunk size.

Reply:
`c_FileUploadDefaultChunkSize ` is only used by File Service, why is it also in this file?

Reply:
I will remove it and move it to file-only constants.